File: 7kyu/shortestWord.js
/*
- return the length of the shortest word(s).
- string will never be empty and you do not need to account for different data types.
*/

function findShort(s) {
    return s.split(" ").sort((a,b) => a.length-b.length)[0].length;
}

console.log(findShort("bitcoin take over the world maybe who knows perhaps")) // 3

File: 7kyu/shortest_word.py
# - return the length of the shortest word(s).
# - string will never be empty and you do not need to account for different data types.

def find_short(s):

    return len(sorted(s.split(" "),key=len)[0])


def find_short(s):
     return min(len(w) for w in s.split())

print(find_short("bitcoin take over the world maybe who knows perhaps")) # 3